src/services/NodeMonitor.ts

```diff
@@ -78,19 +81,21 @@ export class NodeMonitor {
 	};
 
 	private getNodeList = async (): Promise<any> => {
-		// Init fetch node list from config nodes
 		logger.info(`Getting node list`);
 
+		// Fetch node list from config nodes
 		for (const nodeUrl of symbol.NODES) {
-			const peers = await this.fetchNodesByURL(nodeUrl, true);
+			const peers = await this.fetchNodePeersByURL(nodeUrl);
 
 			this.addNodesToList(peers);
 		}
 
 		// Nested fetch node list from current nodeList[]
 		const nodeListPromises = this.nodeList.map(async (node) => {
 			if (isAPIRole(node.roles)) {
-				return this.fetchNodesByURL(getNodeURL(node, monitor.API_NODE_PORT));
+				const hostUrl = await ApiNodeService.buildHostUrl(node.host);
+
+				return this.fetchNodePeersByURL(hostUrl);
```

Comment:
Maybe I miss it, are pulling ``node/peers`` from every api node?

Reply:
Yes, every API node.